packages/clerk-js/src/utils/captcha/getCaptchaToken.ts

```diff
@@ -0,0 +1,26 @@
+import type { CaptchaProvider, CaptchaWidgetType } from '@clerk/types';
+
+import { getHCaptchaToken } from './hcaptcha';
+import { getTunstileToken } from './turnstile';
+
+type CaptchaOptions = {
+  siteKey: string;
+  scriptUrl: string;
+  widgetType: CaptchaWidgetType;
+  invisibleSiteKey: string;
+  captchaProvider: CaptchaProvider;
+};
+
+export const getCaptchaToken = async (captchaOptions: CaptchaOptions) => {
+  const { captchaProvider, widgetType, invisibleSiteKey, siteKey, scriptUrl } = captchaOptions;
```

Comment:
🙃 to avoid copy/paste all the options since they are passThrough you could use:
```typescript
const { captchaProvider, ...captchaProviderOptions } = captchaOptions;
if (captchaProvider === 'hcaptcha') {
   return getHCaptchaToken(captchaProviderOptions);
} else {
// ...
```